### ISSUE TITLE
scaleDropdownField.options is not iterable

### ISSUE THREAD

**Issue by jpike88** [stephanrauh/ngx-extended-pdf-viewer#1389]
https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/f3ff0e3b421af33f92ab3de84de296a0fd45ff0f/projects/ngx-extended-pdf-viewer/src/lib/ngx-extended-pdf-viewer.component.ts#L1967

I use the customToolbar property, which only has buttons like so, excluding the scale dropdown field. This is on purpose.

![Screen Shot 2022-05-26 at 01 43 54](https://user-images.githubusercontent.com/9585787/170344806-340b05b9-1e7b-4d9c-8a50-8209b585bdd4.png)

However,  the field gets put in my DOM like this:

`<span id="scaleSelect" class="invisible dummy-component"></span>`

It's causing the code I referenced above to have a false positive, thinking the scale dropdown is present when it really isn't.

Current workaround is to have this when `pagesLoaded` event fires:

```
		// remove the dropdown scale from the DOM
		document.querySelector('#scaleSelect').remove();
```

I'd make a PR, but it would be a quick fix (just checking to see if `.dummy-component` class is against that element), not sure if you want something more sophisticated.

Thanks

**Comment by stephanrauh** [comment]
By the way, thanks a lot for offering a PR! My approach is even simpler. I just forgot to commit and push my changes.

**Comment by stephanrauh** [comment]
I've just shipped version 13.5.1. I'm closing this issue now.
